File: modules/bluetooth-hid-profile/src/BluetoothHidProfileModule.ts
import { NativeModule, requireNativeModule } from 'expo';

import { BluetoothHidProfileModuleEvents } from './BluetoothHidProfile.types';

declare class BluetoothHidProfileModule extends NativeModule<BluetoothHidProfileModuleEvents> {
  getTheme: () => string;
}

// This call loads the native module object from the JSI.
export default requireNativeModule<BluetoothHidProfileModule>('BluetoothHidProfile');

File: modules/bluetooth-hid-profile/android/src/main/java/expo/modules/bluetoothhidprofile/BluetoothHidProfileModule.kt
package expo.modules.bluetoothhidprofile

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class BluetoothHidProfileModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("BluetoothHidProfile")

    Function("getTheme") {
      return@Function "system"
    }
  }
}
